docs-chef-io/content/server/api_chef_server.md

```diff
@@ -2432,9 +2271,7 @@ The response is similar to:
 
 #### POST
 
-The `POST` method is used to add a key for the specified client.
-
-This method has no parameters.
+The `POST` method is used to add a key for the specified client. This method has no parameters.
```

Comment:
We should keep all the text saying "This method has no parameters." on a separate line.